test/test_component/test_component.vcxproj

```diff
@@ -384,6 +384,8 @@
     <ClCompile Include="Generated Files\module.g.cpp" />
     <ClCompile Include="module.cpp" />
     <ClCompile Include="Optional.cpp" />
+    <ClCompile Include="OverloadClass.cpp" />
+    <ClCompile Include="test_overload.cpp" />
```

Comment:
[mostly preexisting; no action needed] This component has inconsistent naming for files.  Some are snake_case and some are PascalCase.

Reply:
I believe the ones with PascalCase are WinRT classes while the snake_case aren't.

Reply:
Ah, that makes some sense.  I overlooked the pattern.